test/performance/odp_dmafwd.c

```diff
@@ -35,6 +36,7 @@ enum {
 #define DEF_CNT 32768U
 #define DEF_LEN 1024U
 #define DEF_WORKERS 1U
+#define DEF_TIME 10U
```

Comment:
A matter of taste, but since this is a forwarding application I'd set the default value to zero like with l2fwd.

Reply:
Zero in v3.